utilities/rbac.py

```
@@ -0,0 +1,101 @@
+    ) as cluster_role:
+        with create_role_binding(
+            client=client,
+            name=binding_name,
+            namespace=namespace,
+            subjects_kind=subjects_kind,
+            subjects_name=subjects_name,
+            subjects_api_group=subjects_api_group,
+            subjects_namespace=subjects_namespace,
+            role_ref_kind=cluster_role.kind,
+            role_ref_name=cluster_role.name,
+        ):
```

Ruff (0.16.2)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)